### PR TITLE
Issue #175 : Auto determine the address of the Registry based on the Network Selected

### PR DESCRIPTION
**Pull request by anandrgit**
Based on the network selected ( kovan /ropsten/local/main)  , we identify the network ID it is mapped to , using this network ID , the registry address will be read from the networks/Registry.json file .
With this change we get away from manually setting the registry and the ethereum end point in the daemon configuration.( snetd.config.json)

Take Registry address from platform-contracts by default. beta
#175

### CODE REVIEW

File: config/blockchain_network_config.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    "ethereum_json_rpc_endpoint":"https://mainnet.infura.io",
+    "network_id":"1"
+  }
+}`
+	BlockChainNetworkFileName  = "blockchain_network_config.json"
+	EthereumJsonRpcEndpointKey = "ethereum_json_rpc_endpoint"
+	NetworkId                  = "network_id"
+	RegistryAddressKey         = "registry_address_key"
+	//Read this from a config file path todo
+	RegistryJsonFileName = "resources/blockchain/node_modules/singularitynet-platform-contracts/networks/Registry.json"
```
Comment (vsbogd) [inline config/blockchain_network_config.go]: Not sure why this file would be readable? It is a long filename, file as far as I understand should be inside NPM package. How will this path be available from daemon working directory?

File: config/blockchain_network_config.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+)
+
+var networkSelected *NetworkSelected
+
+func determineNetworkSelected(data []byte) {
+	dynamicBinding := map[string]interface{}{}
+	networkName := GetString(BlockChainNetworkSelected)
+	err := json.Unmarshal(data, &dynamicBinding)
+	if err != nil {
+		panic(fmt.Sprintf("cannot parse the JSON configuation file %v for the network %v  to determine the network selected: %v",
```
Comment (vsbogd) [inline config/blockchain_network_config.go]: I would replace `panic` calls here and below by returning `err` to the caller and further and at the end return it from `setAndValidateBlockChainNetworkDetails` and from `Validate` function in `config.go`. Thus all validation errors will be handled by similar way but I don't insist on it.

File: config/blockchain_network_config.go (hunk 1 above)
Comment (anandrgit) [reply config/blockchain_network_config.go]: you are right , will fix this post Beta , closing this PR for now

File: config/blockchain_network_config.go (hunk 2 above)
Comment (anandrgit) [reply config/blockchain_network_config.go]: Will do

**Pull request closed without merging**